jax/_src/interpreters/batching.py

```diff
@@ -134,6 +134,10 @@ def move_axis(ax):
     new_ragged_axes = [(move_axis(ax), sizes) for ax, sizes in self.ragged_axes]
     return RaggedAxis(dst, new_ragged_axes)
 
+  def transpose_ragged_axes(self, perm):
```

Comment:
IMO module-level functions are nicer than methods (for this and `move_stacked_axis`) but nbd!

Reply:
Will move in a future PR.